### PR TITLE
User results

### PR DESCRIPTION
**Pull request by TheoBoucher38**

### CODE REVIEW

File: public/assets/css/style.css [hunk 1]
```diff
@@ -553,3 +553,10 @@ h2 {
 #resultCategories:hover {
     background-color: var(--bs-green-darker);
 }
+<<<<<<< HEAD
```
Comment (aldaitzwild) [inline public/assets/css/style.css]: Unmanaged conflict

**Review by aldaitzwild — CHANGES_REQUESTED** [review]

File: public/assets/css/style.css (hunk 1 above)
Comment (TheoBoucher38) [reply public/assets/css/style.css]: MISS CLICK

**Review by aldaitzwild — CHANGES_REQUESTED** [review]
You have a submodule here too and conflicts with dev

**Comment by aldaitzwild** [comment]
No worthy of a PR

**Pull request closed without merging**